### ISSUE TITLE
`multisearch` continues past "no query signatures loaded, exiting" message

### ISSUE THREAD

**Issue by ctb** [sourmash-bio/sourmash_plugin_branchwater#280]
when I run:
```
sourmash scripts multisearch zip-without-manifest.zip zip-with-manifest.zip -o zzz.csv
```

I get:
```
=> sourmash_plugin_branchwater 0.9.3.dev0; cite Irber et al., doi: 10.1101/2022.11.02.514947

ksize: 31 / scaled: 1000 / moltype: DNA / threshold: 0.01
searching all sketches in 'zip-without-manifest.zip' against 'zip-with-manifest.zip' using 8 threads
Reading query(s) from: 'zip-without-manifest.zip'
Sketch loading error: No such file or directory (os error 2)
WARNING: could not load sketches from path 'PK'
No valid signatures found in query pathlist 'zip-without-manifest.zip'
WARNING: 1 query paths failed to load. See error messages above.
No query signatures loaded, exiting.
Reading search(s) from: 'zip-with-manifest.zip'
Loaded 1 search signature(s)
DONE. Processed 0 comparisons
...multisearch is done! results in 'zzz.csv'
```

it seems a little weird it tells me that "No query signatures loaded, exiting." and then ...keeps on going.

What's even weirder is that I can't figure out where the message is coming from - the strong "No query signatures loaded" doesn't seem to show up in either the branchwater plugin repo, or the sourmash repo!?

**Comment by ctb** [comment]
ahh, found it in `utils.rs`:
```rust
    // Validate sketches                                                        
    if collection.is_empty() {
        eprintln!("No {} signatures loaded, exiting.", report_type);
        return Ok(());
    }
    eprintln!("Loaded {} {} signature(s)", collection.len(), report_type);
    Ok(())
```

**Comment by bluegenes** [comment]
Right, this is related to https://github.com/sourmash-bio/sourmash_plugin_branchwater/issues/207.

I can make a new issue to better reflect the new goal, which is to actually err from here, rather than returning Ok.

**Comment by ctb** [comment]
I was thinking of linking that issue in, but I feel like this is slightly different, because this is a specific error that shouldn't be bypassed. I'll muse more.

**Comment by ctb** [comment]
There's actually two different problems here - one is the confusion from the printed out messages during `load_collection`, and the other is that `multisearch` continues past `load_sketches`, when `load_sketches` ends up returning 0 actual sketches that match (b/c of selection, or whatever).

Tackling this in 430.

**Comment by ctb** [comment]
* Fixed in #437, which will be part of #430.
